bundles/ui/org.openhab.ui.webapp/src/main/java/org/openhab/ui/webapp/internal/render/ChartRenderer.java

```diff
@@ -62,7 +62,7 @@ public boolean canRender(Widget w) {
 
 			if(chart.getRefresh()>0) {
 				snippet = StringUtils.replace(snippet, "%setrefresh%", "<script type=\"text/javascript\">imagesToRefreshOnPage=1</script>");
-				snippet = StringUtils.replace(snippet, "%refresh%", "id=\"%id%\" onload=\"setTimeout('reloadImage(\\'%url%\\', \\'%id%\\')', " + chart.getRefresh() + ")\"");
```

Comment:
When I last time looked in the code **reloadImage** function was meant to be global. But due to syntax error it is not (It is in some browsers). I would suggest to fix this **reloadImage** function's globalness instead of modifying template.
I may be wrong...